forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -3372,6 +3371,7 @@ public final void runLeavesPlayCommands() {
             c.run();
         }
         leavePlayCommandList.clear();
+        clearRingBearer();
```

Comment:
These two calls should no longer be needed because the GameCommand already runs them?